types/knockout/index.d.ts

```diff
@@ -4,21 +4,23 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 
-interface KnockoutSubscribableFunctions<T> {
+interface KnockoutExtensionFunctions {
     [key: string]: KnockoutBindingHandler | undefined;
+}
 
-	notifySubscribers(valueToWrite?: T, event?: string): void;
+interface KnockoutSubscribableFunctions<T> {
+    notifySubscribers(valueToWrite?: T, event?: string): void;
 }
+type KnockoutSubscribableExtensionFunctions<T> = KnockoutSubscribableFunctions<T> & KnockoutExtensionFunctions;
```

Comment:
I tried that, but it created problems with the typings for knockout.deferred.updates.